### PR TITLE
Add nix build configs

### PR DESCRIPTION
**Pull request by NickLarsenNZ**
This makes it easier for users of Nix/NixOS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,28 +6,59 @@ Version: 0.0.9
...
+```nix
+{ lib, pkgs, ... }:
+with lib;
+let
+  beku = (import (pkgs.fetchFromGitHub {
+    owner = "stackabletech";
+    repo = "beku.py";
+    rev = "062defa4da2ec504c38d3a21916e871fd95d03f6"; # commit hash
+    hash = "sha256-Oq8BhByYDptD84551Rodi6T7MwI8e/6dxIX7p0xNK+A="; # use lib.fakeHash to find new hashes when upgrading
+  }) {}).beku.build;
```
[the remark this answers is not in the document]
Comment (NickLarsenNZ) [reply README.md:33]: Actually, I can test this now that the changes are committed. And I'm quite certain `build` is just an _item_ (I don't know the nix nomenclature) meaningful in stackable-cockpit when building the crates).

Will post an update in a moment.

[the remark this answers is not in the document]
Comment (nightkr) [reply README.md:33]: It's just what the attribute happens to be named in that repository.. here you'd do `beku = (import ...).beku`.

That said, when using it from a nix-shell like this, arguably you should do:

```nix
let
  beku = pkgs.callPackage (pkgs.fetchFromGitHub { ... } + "/beku.nix") {};
in ...
```

[the remark this answers is not in the document]
Comment (NickLarsenNZ) [reply README.md]: Tested locally, and fixed in e7e81725040d0a974f0010308091d4c194cd76ea

[the remark this answers is not in the document]
Comment (NickLarsenNZ) [reply README.md]: > It's just what the attribute happens to be named in that repository.. here you'd do `beku = (import ...).beku`.
> 
> That said, when using it from a nix-shell like this, arguably you should do:
> 
> ```nix
> let
>   beku = pkgs.callPackage (pkgs.fetchFromGitHub { ... } + "/beku.nix") {};
> in ...
> ```

Sorry, just saw this... 

Then I guess the instructions I wrote are for NixOs only.
For a dev-shell (which was my whole reason for this), `shell.nix` would be separate instructions. I will fix that now.

[the remark this answers is not in the document]
Comment (nightkr) [reply README.md]: The same applies to both NixOS and shell.nix.

[the remark this answers is not in the document]
Comment (NickLarsenNZ) [reply README.md]: Split docs in 9ec4235b217d4e1fe195890dfb2a843caf094e04

[the remark this answers is not in the document]
Comment (NickLarsenNZ) [reply README.md]: Unified docs in: fad6b341939b747312c4fc1dfa2eaaa1093f4495

**Pull request merged**